2020-12-22 23:10:21,804 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:10:39,900 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:10:43,656 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:14:03,596 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:14:06,813 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:14:17,355 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:14:20,066 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:15:20,167 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:15:23,142 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:17:01,788 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:17:04,724 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:22:13,075 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:22:15,762 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:22:30,771 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:22:33,661 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:29:32,596 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:29:41,424 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:29:48,949 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:29:51,770 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:31:01,913 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:31:04,956 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:32:11,183 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:32:14,367 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:32:24,913 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:32:28,515 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:12,501 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:15,791 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:22,701 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:29,336 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:32,683 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:39,922 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:35:43,566 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:36:06,786 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:36:14,377 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:36:17,735 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:36:54,204 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:36:57,549 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:14,581 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:21,009 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:23,749 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:37,626 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:40,415 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:37:56,017 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:38:01,204 ERROR: Exception on /fetch [POST] [in C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\44738\OneDrive\Documents\Web\app\main\routes.py", line 59, in fetch
    group = ContactGroups.query.filter_by(group_name = group_name).first()
NameError: name 'ContactGroups' is not defined
2020-12-22 23:39:10,590 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
2020-12-22 23:39:16,083 ERROR: Exception on /fetch [POST] [in C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\44738\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\44738\OneDrive\Documents\Web\app\main\routes.py", line 59, in fetch
    group = group.query.filter_by(group_name = group_name).first()
UnboundLocalError: local variable 'group' referenced before assignment
2020-12-22 23:39:56,240 INFO: Project startup [in C:\Users\44738\OneDrive\Documents\Web\app\__init__.py:66]
